translate/src/api/comment.ts

```diff
@@ -0,0 +1,59 @@
+import { GET, POST } from './_base';
```

Comment:
File names starting with `_` are a new pattern within the `translate` app I believe.

Care to explain when do we use it in the `README`?